scripts/ready-prs.py

```
@@ -330,17 +328,39 @@ def phase_kick_ci(prs: list[PRInfo], runs: list[dict], ctx: Ctx) -> int:
+                continue
             except RuntimeError:
-                print(f"{RED}✗ failed{RESET}")
-        else:
-            print(f"{YELLOW}no CI run to re-run (empty commit may be needed){RESET}")
+                pass  # Fall through to empty commit
+        # No CI run exists for the current head — push an empty commit to trigger one.
+        try:
+            gh_text("api", f"repos/{ctx.repo_slug}/git/refs/heads/{pr.branch}",
+                     "-X", "PATCH", "-f", f"sha={pr.head_sha}", "--silent")
+            # Push empty commit via the API
+            subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)